### ISSUE TITLE
Support for params with structured_value of type list

### ISSUE THREAD

**Issue by rhudz-ingram** [cloudblue/connect-devops-testing-library#31]
Hi.
Could you add support for list type to the `_param_members()` method here?
https://github.com/cloudblue/connect-devops-testing-library/blob/master/connect/devops_testing/request.py#L66

As I've tested in Connect sandbox, creating a configuration param of type Object with `structured_value` containing a list.
Then getting a request by API I have this in `asset -> configuration -> params`:
```
{
"id": "offline_mode_list",
"structured_value": [
  "a1",
  "a2"
],
}
```

Thanks.

**Comment by othercodes** [comment]
Hi @rhudz-ingram what is the parameter type that you want to handle as a list?

**Comment by othercodes** [comment]
Hi @rhudz-ingram please check #34

**Comment by othercodes** [comment]
Solved in #34.
